package.json

```diff
@@ -7,8 +7,10 @@
     "node": "18.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc",
+    "dev": "nodemon --exec ts-node src/index.ts",
+    "start": "node dist/index.js",
+    "test": "jest"
```

Comment:
I changed what was `start` script to be `dev` script because in production environment "start" is the execution command, dev is more for local development- we need to figure out how to invoke the build step as part of the ci/cd pipeline.